### PR TITLE
Settings theme issue fixed

### LINKED ISSUE #5: Settings theme does not match with app's theme
**Describe the bug**
White text is used over black background in Settings while its opposite in other screens of the app

**To Reproduce**
1. Go to Settings screen
2. Observe the difference

**Expected behavior**
Black text should be used in white background.

**Screenshots**
settings_theme

**Smartphone (please complete the following information):**
 - Device: Redmi 4
 - OS: Android Nougat

### PR DESCRIPTION
**Pull request by rob729**
Fixes #5 
Screenshot_2019-07-01-12-09-21-553_com iitdh sonusourav instigo

### CODE REVIEW

**Comment by rob729** [comment]
@sonusourav Please review this PR.

**Comment by sonusourav** [comment]
Change the switch color when it is in off state and use some other dark color for secondary color

**Comment by rob729** [comment]
@sonusourav Please specify the color.

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -11,6 +11,13 @@
     <item name="android:actionBarTabTextStyle">@color/white</item>
   </style>
 
+  <style name="SettingsTheme" parent="Theme.AppCompat.NoActionBar">
```
Comment (sonusourav) [inline app/src/main/res/values/styles.xml]: use text color for secondary color and put these added lines in the last. Rest 👌

**Review by sonusourav — CHANGES_REQUESTED** [review]

**Comment by rob729** [comment]
@sonusourav I have done the requested changes.

**Review by sonusourav — APPROVED** [review]

**Pull request merged**